airflow/migrations/versions/0051_3_0_0_update_task_instance_trigger_timeout_to_utcdatetime.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+update trigger_timeout column in task_instance table to UTC.
+
+Revision ID: 038dc8bc6284
+Revises: e229247a6cb1
+Create Date: 2024-11-30 10:47:17.542690
+
+"""
+
+from __future__ import annotations
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+import airflow
+
+# revision identifiers, used by Alembic.
+revision = "038dc8bc6284"
+down_revision = "e229247a6cb1"
+branch_labels = None
+depends_on = None
+airflow_version = "3.0.0"
+
+
+def upgrade():
+    """Apply update task instance trigger timeout to utcdatetime."""
+    conn = op.get_bind()
+    dialect = conn.dialect.name
+    if dialect == "postgresql":
+        with op.batch_alter_table("task_instance", schema=None) as batch_op:
+            batch_op.alter_column(
+                "trigger_timeout",
+                existing_type=postgresql.TIMESTAMP(),
+                type_=airflow.utils.sqlalchemy.UtcDateTime(timezone=True),
+                existing_nullable=True,
+            )
```

Comment:
Hmm why this for Postgres?

You could just use `from airflow.migrations.db_types import TIMESTAMP`

https://github.com/apache/airflow/blob/5079f7d8557ac2334aadfdbb9ce209199116fb06/airflow/migrations/db_types.py#L68

Reply:
cc @ephraimbuddy our in-house DB migration expert

Reply:
For reference: Ephraim fixed a lot of inconsistencies between ORM and DB migrations in https://github.com/apache/airflow/pull/44221

Reply:
I didn't see the mention until today. Looking...

Reply:
Yes. You should remove the 'if postgresql' because this should affect all DBs and also you should import TIMESTAMP from db_types. `from airflow.migrations.db_types import TIMESTAMP`. See #44221 where I had to import and use it too

Reply:
But we want to now change it to a datetime with timezone info, so we need to update all the DBs to use the timezone info